frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -56,8 +60,13 @@ const AdminDashboard = (): React.ReactElement => {
 
   useEffect(() => {
     setIsLoading(true);
-    reviewAPIClient.getReviews().then((allReviews: ReviewResponse[]) => {
-      setData(mapReviewResponseToReview(allReviews));
+    reviewAPIClient.getReviews().then((resp: RetrieveReviewResponse) => {
```

Comment:
This seems to only fetch the first 10 reviews. I think we need to either implement pagination on the admin table (preferred) or set the default behaviour in the backend to fetch all reviews by default (ie default page size is num of total reviews)